features/step_definitions/networking.rb

```diff
@@ -790,3 +790,13 @@
   @result = _admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.type}")
   raise "the networkType is not #{network_type}" unless @result[:response] == network_type
 end
+
```

Comment:
This is useful definition and can be used in future as well. We can identify INTERNAL IP of any node using this.